File: src/test/samples/invalidSamples/invalidSample.rb
#region FirstRegion
x = 42
#endregion
 
  # endregion Invalid end boundary
  # region Invalid start boundary

#   region Second Region
class MyClass
    #  region    InnerRegion
    def my_method
    end
    # endregion   ends InnerRegion

    #region
    def my_method2
    end
    #   endregion
end
# endregion

File: src/test/samples/invalidSamples/invalidSample.fs
//#region   FirstRegion
let x = 42
//#endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

// #region Second Region
type MyClass() =
    // #region InnerRegion
    member this.MyMethod() = ()
    // #endregion ends InnerRegion

    // #region
    member this.MyOtherMethod() = ()
    //        #endregion
// #endregion

